src/gproc_pool.erl

```diff
@@ -557,32 +556,20 @@ execute_claim(F, K, Pid) ->
 setup_wait(nowait, _) ->
     nowait;
 setup_wait({busy_wait, MS}, Pool) ->
```

Comment:
```suggestion
setup_wait({busy_wait, MS}, _Pool) ->
```

Reply:
@uwiger thanks for the review. Now fixed and force-pushed.